src/routes/items/+page.server.ts

```diff
@@ -21,13 +21,13 @@ export const actions = {
         if (!userId) {
             throw redirect(302, "/unauthorised");
         }
+        var createdItem = null;
```

Comment:
instead of var lets use `let`. Its just less problematic https://stackoverflow.com/questions/762011/what-is-the-difference-between-let-and-var

Reply:
done, changed to `let` and `const` where appropriate